ingestion/src/metadata/ingestion/ometa/client.py

```diff
@@ -142,7 +142,7 @@ def _request(
             headers = {"Content-type": "application/json"}
         base_url = base_url or self._base_url
         version = api_version if api_version else self._api_version
-        url: URL = URL(base_url + "/" + version + path)
+        url: URL = URL(base_url + version + path)
```

Comment:
trying to fix metabase with this, since we'd be adding double `//` from `AnyUrl` strings. Not sure if it's the safest way

Reply:
UPDATE: we'll need to fix this directly in the metabase client